Sources/SnapshotTesting/SnapshotTesting.swift

```diff
@@ -116,3 +108,57 @@ private var trackSnapshots = {
     print("Found \(count) stale snapshot\(count == 1 ? "" : "s"):\n\n\(list)")
   }
 }()
+
+public protocol SnapshotStringConvertible {
```

Comment:
We now have a protocol that we can use to customize the snapshot of types. It takes precedence over `CustomStringConvertible` and `CustomDebugStringConvertible`.